app/templates/index-g-cloud.html

```diff
@@ -25,7 +25,7 @@
   </div>
   <ul class="browse-list">
     <li class="browse-list-item">
-      <a href="/search?q=&lot=saas" class="browse-list-item-link">
+      <a href="/search/?q=&lot=saas" class="browse-list-item-link">
```

Comment:
Not directly related to this PR, I know, but is the empty `q=` param required here?